cargo-dist/src/init/v0.rs

```diff
@@ -1120,6 +1124,20 @@ fn apply_dist_to_metadata(metadata: &mut toml_edit::Item, meta: &DistMetadata) {
         *github_attestations,
     );
 
+    apply_string_or_list(
+        table,
+        "github_attestations-filters",
```

Comment:
Looks like a typo in the name here - using an underscore for the first separator instead of a dash. This would make `init` generate an incorrect config.
```suggestion
        "github-attestations-filters",
```